frontend/mirage/config.js

```diff
@@ -65,7 +65,7 @@ export default function() {
     };
   });
 
-  
+
   /**
```

Comment:
Lots of little space things like this, is this intentional?

Reply:
Definitely not intentional, been trying to remove it and haven't succeeded